pkg/triage/github.go

```diff
@@ -45,3 +47,11 @@ func MustReadToken(path string, env string) string {
 	}
 	return token
 }
+
+func MustCreateGithubClient(githubAPIRawURL string, httpClient *http.Client) (*github.Client, error) {
+	if githubAPIRawURL != "" {
+		// passing base url to uplaod URL is safe because triage-party does NOT upload anything.
```

Comment:
the comment isn't really necessary, but if you want to keep it, fix the spelling of "upload". Thanks!

Reply:
OK. I'll remove it.